src/routes/(app)/+layout.server.ts

```diff
@@ -131,6 +132,28 @@ export async function load(params) {
 			  ).then((res) => filterUndef(res))
 			: null
 	]);
+	let cmsAgewall = null;
+	const session = await collections.sessions.findOne({
+		sessionId: params.locals.sessionId
+	});
```

Comment:
You should add the `acceptAgeLimitation` directly in `params.locals.acceptAgeLimitation` in `hooks.server.ts`, to avoid querying the DB once more.

eg near here:

```
		event.locals.email = session.email;
		event.locals.npub = session.npub;
		event.locals.sso = session.sso;
```